server.go

```diff
@@ -1,15 +1,18 @@
 package main
 
 import (
-    "net/http"
-    "github.com/labstack/echo"
-    "github.com/labstack/echo/engine/standard"
+	"github.com/labstack/echo"
+	"github.com/wimpykid26/hermes/api"
+	"net/http"
 )
 
 func main() {
-    e := echo.New()
-    e.GET("/", func(c echo.Context) error {
-        return c.String(http.StatusOK, "Hello, World!")
-    })
-    e.Run(standard.New(":1323"))
-}
\ No newline at end of file
+	e := echo.New()
+	e.GET("/insert_user", api.Insert_user)
```

Comment:
Lets divide api into, api/users.go, api/events.go, api/hubs.go, and similarly, keep the routes as, `/api/user/create` etc.